scripts/generate_sphinx_docs.sh

```diff
@@ -55,41 +55,9 @@ function build_python_source_in_docker() {
     popd
 }
 
-#######################################
```

Comment:
I think that you need to delete this whole `.sh` file.  Also is this script referenced anywhere else?  Other scripts, documentations?  Do they need to be updated?

Reply:
Yes, the whole script can be removed. Deleted script and updated references. Time saved is 30 seconds (already added in the description).